src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1104,6 +1105,13 @@
 	end
 
 
+	function m.linkObjects(fcfg, condition)
+		if fcfg.linkbuildoutputs == false then
+			m.element("LinkObjects", condition, "false")
```

Comment:
You meant `fcfg.linkbuildoutputs ~= nil`, right? Done. I added some tests while I was at it.